pretix_sepadebit/payment.py

```diff
@@ -142,6 +173,13 @@ def payment_is_valid_session(self, request):
             request.session.get('payment_sepa_bic', '') != ''
         )
 
+    def settings_form_clean(self, cleaned_data):
+        super().settings_form_clean(cleaned_data)
+
+        if cleaned_data['payment_sepadebit_earliest_due_date']:
+            if not (cleaned_data['payment_sepadebit_mail_payment_reminder_subject'] and cleaned_data['payment_sepadebit_mail_payment_reminder_text']):
```

Comment:
Avoid a crash if these fields are missing from cleaned_data
```suggestion
        if cleaned_data.get('payment_sepadebit_earliest_due_date'):
            if not (cleaned_data.get('payment_sepadebit_mail_payment_reminder_subject') and cleaned_data.get('payment_sepadebit_mail_payment_reminder_text')):
```

Reply:
Also, I think it might be nice to provide a default text for these emails? You can do so by adding defaults to your signals.py file:


```
settings_hierarkey.add_default(
    "payment_sepadebit_mail_payment_reminder_text",
    LazyI18nString.from_gettext(
        gettext_noop(
            "Hello,\n\n"
            "you ordered a ticket for {event}.\n\n"
            "We will debit your bank account {iban} on or shortly after {due_date}. The payment will appear on your bank statement as {creditor_name} with reference {reference} and creditor identifier {creditor_id}.\n\n"
            "You can change your order details and view the status of your order at\n"
            "{url}\n\n"
            "Best regards,\n"
            "Your {event} team"
        )
    ),
    LazyI18nString,
)
```